### PR TITLE
rbac: simplify and make authorization HTTP/TCP tasks consistent

### PR DESCRIPTION
**Pull request by yangminzhu**
* Simplify the Bookinfo deployment, the service accounts could just be
added with the default Bookinfo deployment. (Depending on https://github.com/istio/istio/pull/14678 to be merged to 1.2)

* Make the `Before you begin` section more consistent for HTTP and TCP
tasks

### CODE REVIEW

**Comment by yangminzhu** [comment]
@liminw @rcaballeromx Could you take a look? This is not for bug fix but simplify the task so that user don't need to deploy bookinfo twice for the authz task.

File: content/docs/tasks/security/authz-http/index.md [hunk 1]
```diff
@@ -18,29 +18,19 @@ The activities in this task assume that you:
 
 * Read the [authorization concept](/docs/concepts/security/#authorization).
 
-* Follow the instructions in the [Kubernetes quick start](/docs/setup/kubernetes/install/kubernetes/) to
-install Istio **with mutual TLS enabled**.
+* Follow the [Kubernetes quick start](/docs/setup/kubernetes/install/kubernetes/) to install Istio **with the strict mutual TLS profile**.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authz-http/index.md]:
```suggestion
* Follow the [Kubernetes quick start](/docs/setup/kubernetes/install/kubernetes/) to install Istio using the **strict mutual TLS profile**.
```

File: content/docs/tasks/security/authz-http/index.md [hunk 2]
```diff
@@ -18,29 +18,19 @@ The activities in this task assume that you:
...
 
-* Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+* Deploy the [Bookinfo](/docs/examples/bookinfo#deploying-the-application) sample application.
 
-* Create service accounts for the Bookinfo application. Run the following command to create service
-account `bookinfo-productpage` for `productpage` and service account `bookinfo-reviews` for `reviews`:
+After deploying the Bookinfo application, go to the Bookinfo product page at `http://$GATEWAY_URL/productpage`. On
+the product page, you can see:
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authz-http/index.md]:
```suggestion
the product page, you can see the following sections:
```

File: content/docs/tasks/security/authz-http/index.md [hunk 3]
```diff
@@ -18,29 +18,19 @@ The activities in this task assume that you:
...
-
-{{< tip >}}
-If you are using a namespace other than `default`, use `kubectl -n namespace ...` to specify the namespace.
-{{< /tip >}}
-
-* Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). You should see:
-
-    * The "Book Details" section in the lower left part of the page, including type, pages, publisher, etc.
-    * The "Book Reviews" section in the lower right part of the page.
+* The **Book Details** section on the lower left of the page includes book type, number of
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authz-http/index.md]:
```suggestion
* **Book Details** on the lower left side, which includes: book type, number of
```

File: content/docs/tasks/security/authz-http/index.md [hunk 4]
```diff
@@ -18,29 +18,19 @@ The activities in this task assume that you:
...
+  pages, publisher, etc.
+* The **Book Reviews** section on the lower right of the page.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authz-http/index.md]:
```suggestion
* **Book Reviews** on the lower right of the page.
```

File: content/docs/tasks/security/authz-tcp/index.md [hunk 5]
```diff
@@ -132,91 +112,59 @@ define access control policies to grant access to the MongoDB service.
...
-        {{< /text >}}
-
-    * To deploy in a cluster **without** automatic sidecar injection enabled:
-
-        {{< text bash >}}
-        $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo-ratings-v2.yaml@)
-        {{< /text >}}
-
-    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).  You should see:
+Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).  You should see:
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authz-tcp/index.md]: Use the same wording as you used on the previous files.

File: content/docs/tasks/security/authz-tcp/index.md [hunk 6]
```diff
@@ -132,91 +112,59 @@ define access control policies to grant access to the MongoDB service.
...
-    $ kubectl apply -f @samples/bookinfo/platform/kube/rbac/mongodb-policy.yaml@
-    {{< /text >}}
-
-    The step above does the following:
+{{< text bash >}}
+$ kubectl apply -f @samples/bookinfo/platform/kube/rbac/mongodb-policy.yaml@
+{{< /text >}}
 
-    * Creates a service role "mongodb-viewer" which allows access to the port 27017 of the MongoDB service.
+The step above does the following:
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authz-tcp/index.md]:
```suggestion
Once applied, the policy has the following effects:
```

File: content/docs/tasks/security/authz-tcp/index.md [hunk 7]
```diff
@@ -132,91 +112,59 @@ define access control policies to grant access to the MongoDB service.
...
+
+* Creates a service role "mongodb-viewer" which allows access to the port 27017 of the MongoDB service.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authz-tcp/index.md]:
```suggestion
* Creates the following `mongodb-viewer` service role, which allows access to the MongoDB service on port 27017.
```

File: content/docs/tasks/security/authz-tcp/index.md [hunk 8]
```diff
@@ -132,91 +112,59 @@ define access control policies to grant access to the MongoDB service.
...
+      rules:
+      - services: ["mongodb.default.svc.cluster.local"]
+        constraints:
+        - key: "destination.port"
+          values: ["27017"]
+    {{< /text >}}
 
-        {{< text yaml >}}
-        apiVersion: "rbac.istio.io/v1alpha1"
+* Creates a service role binding `bind-mongodb-viewer` which assigns the `mongodb-viewer` role to `bookinfo-ratings-v2`.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authz-tcp/index.md]:
```suggestion
* Creates the following `bind-mongodb-viewer` service role binding, which assigns the `mongodb-viewer` role to the `bookinfo-ratings-v2` service.
```

File: content/docs/tasks/security/authz-permissive/index.md [hunk 9]
```diff
@@ -8,7 +8,7 @@ keywords: [security,access-control,rbac,authorization]
 The [authorization permissive mode](/docs/concepts/security/#authorization-permissive-mode) allows
 you to verify authorization policies before applying them in a production environment.
 
-The authorization permissive mode is an experimental feature in version 1.1. Its interface can change
+The authorization permissive mode is an experimental feature in version 1.2. Its interface can change
```
[the remark this answers is not in the document]
Comment (yangminzhu) [reply content/docs/tasks/security/authz-permissive/index.md]: I was wondering should we update the version to make it clear it's still experimental in 1.2

**Comment by yangminzhu** [comment]
@liminw @rcaballeromx Comments addressed, PTAL, thanks.

File: content/docs/tasks/security/authz-http/index.md [hunk 10]
```diff
@@ -18,29 +18,19 @@ The activities in this task assume that you:
 
 * Read the [authorization concept](/docs/concepts/security/#authorization).
 
-* Follow the instructions in the [Kubernetes quick start](/docs/setup/kubernetes/install/kubernetes/) to
-install Istio **with mutual TLS enabled**.
+* Follow the [Kubernetes quick start](/docs/setup/kubernetes/install/kubernetes/) to install Istio using the **strict mutual TLS profile**.
 
-* Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+* Deploy the [Bookinfo](/docs/examples/bookinfo#deploying-the-application) sample application.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/authz-http/index.md]:
```suggestion
* Deploy the [Bookinfo](/docs/examples/bookinfo/#deploying-the-application) sample application.
```

Comment (yangminzhu) [reply content/docs/tasks/security/authz-http/index.md]: Thanks, updated.

**Pull request merged**